app/components/public/stream/join-video.ts

```diff
@@ -32,7 +33,9 @@ export default class JoinVideo extends Component<Args> {
     this.canAccess = can_access;
   }
 
+  @action
   openPanel(): void {
+    this.args.showSidePanel();
```

Comment:
What if showSidePanel is not passed?

What if user does not have access to the videos?